### DISCUSSION TITLE
GOLFZON Wave Support

### DISCUSSION THREAD

**Discussion by RaffoNin** [springbok/MLM2PRO-GSPro-Connector#120] [Ideas]
Been trying to use the OCR on the connector and it works. Problem is the GOLFZON Wave Skills app doesn't just return the number, it animates to it. For example 25 mph ball speed would animate from 0,1,1.99,2,3 instead of just showing the final number. This means that's if i hit a 25mpg chip it'll show the earlier number like 3.6 mph. 

Is there a way for the OCR to ignore the animation. I can't send a video right now as I currently don't have the Wave with me but here's a screenshot of the app. 

![image](https://github.com/user-attachments/assets/2e52deb5-da9b-48dc-9ceb-6b27939f474e)

**Comment by wonder99** [comment]
Please send the video.  I'd guess that the animation is going to be a problem though, since the OCR is looking for changes.  Even if it could be made to work, of course there'd be a big shot latency, as you'd need to wait for the animation to stop before even sending the shot to GSPRO

**Reply by RaffoNin** [reply]
Sorry for the late reply. Here's a video of the numbers. I'd be ok with delay. If you can add a slider I'd be willing to experiment what delay works for the GOLFZON Wave App!

https://github.com/user-attachments/assets/569356f7-262f-4213-ab0f-af2a2cb3a0f4

**Reply by wonder99** [reply]
I don't see an easy way to support this.  We currently look for changes in the ROI boxes, then convert to numbers.  Here we'd need to detect a change, then detect when changes stop happening and convert those.  I'll let @springbok consider it.

**Comment by bj-neilson** [comment]
@RaffoNin hey did you ever manage to find or create a working solution for GOLFZON Wave with GS Pro?
